curriculum/challenges/english/08-coding-interview-prep/algorithms/implement-selection-sort.english.md

```diff
@@ -41,14 +41,11 @@ tests:
 
 ```js
 function selectionSort(array) {
-  // change code below this line
-
-  // change code above this line
```

Comment:
Please do not remove `  // change code above this line` from the challenge seed code.